addons/gamification/models/gamification_karma_rank.py

```diff
@@ -35,7 +35,7 @@ def _compute_rank_users_count(self):
     @api.model_create_multi
     def create(self, values_list):
         res = super(KarmaRank, self).create(values_list)
-        users = self.env['res.users'].sudo().search([('karma', '>', 0)])
+        users = self.env['res.users'].sudo().search([('karma', '>', res.karma_min)])
```

Comment:
this can be called in batch, `res.karma_min` will crash in that case (maybe you should take the min value ? not 100% sure)